src/components/gallery/HdGallery.vue

```diff
@@ -16,6 +16,7 @@
       <div class="gallery__main">
         <HdGalleryMedia
           :item="currentItem"
+          :link="link"
           :aspect-ratio="aspectRatio"
         />
```

Comment:
You could `v-bind="$attrs"` here and remove `link` and `aspectRatio` from the `HdGallery` props as they are part of `HdGalleryMedia` props.

I'm not super familiar with `HdGallery` so if it might not make totally sense hehehe

Reply:
🐒  hehe, thanks. I'll try use `v-bind`.

Reply:
I tried but it didn't work. I checked with @aym3nb also and it can't dispatch variables from the parent component. Possible reasons;

1. HDGallery component doesn't have a prop as `item` but `HDGalleryMedia` has it. 
2. `item` prop of the HDGallery component has one watcher. I can't watch the parent component's prop in the child component. 

If you have any thoughts about it just let me know.